src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data_clear_experience_configs.py

```diff
@@ -0,0 +1,461 @@
+"""translation data
+
+Revision ID: 14acee6f5459
+Revises: a1e23b70f2b2
+Create Date: 2024-01-09 21:17:13.115020
+
+"""
+
+import uuid
+from enum import Enum
+
+import yaml
+from alembic import op
+from sqlalchemy import text
+from sqlalchemy.engine import ResultProxy
+
+# revision identifiers, used by Alembic.
+
+revision = "14acee6f5459"
+down_revision = "a1e23b70f2b2"
+branch_labels = None
+depends_on = None
+
+
+def generate_record_id(prefix):
+    return prefix + "_" + str(uuid.uuid4())
+
+
+class ExperienceConfigIds(Enum):
+    EEA_PRIVACY_CENTER = 1
+    EEA_MODAL_AND_BANNER = "pri-c8ff-78d6-4a02-850f-2c09dda-over-config"
+    EEA_TCF_OVERLAY = "d7c3ce0a-a3b3-43ff-bf6f-3d8-tcf-over-config"
+    US_MODAL = "pri-76b8-cc52-11ee-9eaa-8ef97a0-over-config"
+    US_PRIVACY_CENTER = "pri-27d4-cc53-11ee-9eaa-8ef97a04-pri-config"
+
+
+class ComponentType(Enum):
+    PrivacyCenter = "privacy_center"
+    Overlay = "overlay"
+    TCFOverlay = "tcf_overlay"
+
+
+EEA_MAPPING = {
+    ComponentType.PrivacyCenter: ExperienceConfigIds.EEA_PRIVACY_CENTER,
+    ComponentType.Overlay: ExperienceConfigIds.EEA_MODAL_AND_BANNER,
+    ComponentType.TCFOverlay: ExperienceConfigIds.EEA_TCF_OVERLAY,
+}
+US_MAPPING = {
+    ComponentType.PrivacyCenter: ExperienceConfigIds.US_PRIVACY_CENTER,
+    ComponentType.Overlay: ExperienceConfigIds.US_MODAL,
+}
+
+NOTICES_TO_EXPERIENCE_CONFIG = {
+    "marketing": EEA_MAPPING,
+    "functional": EEA_MAPPING,
+    "essential": EEA_MAPPING,
+    "analytics": EEA_MAPPING,
+    "data_sales_and_sharing": US_MAPPING,
+}
+
+
+def load_default_experience_configs():
+    """
+    Loads default experience config definitions from yml file.
+
+    Maps the experience config definitions keyed by the ID enum.
+    """
+    raw_experience_config_map = {}
+    experience_config_map = {}
+    with open("privacy_experience_config_defaults.yml", "r") as f:
```

Comment:
argh this doesn't work well if you try to run the migrations manually from the alembic dir, i will either need to embed this data in the file or figure out a way to make alembic be able to read this in either the app startup context or a manual migration context ...